server/opts.go

```diff
@@ -697,7 +697,21 @@ func (o *Options) processConfigFileLine(k string, v interface{}, errors *[]error
 			}
 			o.TrustedOperators = append(o.TrustedOperators, opc)
 		}
+
+		if o.AccountResolver == nil && len(o.TrustedOperators) == 1 {
+			if accUrl, err := parseURL(o.TrustedOperators[0].AccountServerURL, "account resolver"); err == nil {
+				// accommodate nsc which appends "/accounts" during nsc push
+				suffix := ""
+				if accUrl.Path == "/jwt/v1/" || accUrl.Path == "/jwt/v1" {
+					suffix = "/accounts"
+				}
+				o.AccountResolver, _ = NewURLAccResolver(accUrl.String() + suffix)
+			}
+		}
+
 	case "resolver", "account_resolver", "accounts_resolver":
+		o.AccountResolver = nil
```

Comment:
Can you explain why we set it to nil here? Is it because of config reload?

Reply:
I set it to nil to cover the following case:
"operator" then "resolver" 
In this case we'd already have an account resolver.
If now "resolver" is ALSO mis configured, line 743 wouldn't catch an error.

Reply:
Ok, so it is to reset it in case the initialization in this section fails, right?

Reply:
But we would still overwrite the value from "trusted" section..

Reply:
I think it's reasonable to fail if you specified operator and resolver: bad()
Do you?

Reply:
Yes.. based on previous explanation of the "take precedence", I am fine with this. Setting it to nil is so that if the processing of "resolver" fails, we actually report the config error and fail to start.

Reply:
great, will add the comment and squash